airflow/models.py

```diff
@@ -336,7 +336,8 @@ def kill_zombies(self, session):
                 if ti.task_id in dag.task_ids:
                     task = dag.get_task(ti.task_id)
                     ti.task = task
-                    ti.handle_failure("{} killed as zombie".format(ti))
+                    ti.test_mode = False
```

Comment:
@r39132 still do you think that I should revert the changes from `models.py`? As in existing code as well we are doing same (line 338). Also without sending context, handling zombie is just like marking them as `failed`.